src/Umbraco.Core/Configuration/GlobalSettings.cs

```diff
@@ -31,7 +31,7 @@ public class GlobalSettings : IGlobalSettings
         private static string _reservedPaths;
         private static string _reservedUrls;
         //ensure the built on (non-changeable) reserved paths are there at all times
-        internal const string StaticReservedPaths = "~/app_plugins/,~/install/,"; //must end with a comma!
+        internal const string StaticReservedPaths = "~/app_plugins/,~/install/,~/mini-profiler-resources/"; //must end with a comma!
```

Comment:
`//must end with a comma!`

It currently doesn't, also.. why must it, seems weird?

Reply:
some old crappy code, i dunno, i didn't actually change that from v7, in v7 the static reserved paths also need this